Homework2/Program.cs

```diff
@@ -11,9 +11,71 @@
 
 Console.WriteLine();
 bankAccount4.AddMoney(20M);
-Console.WriteLine(bankAccount4);
-bankAccount4.PullOfMoney(50M);
-bankAccount4.PullOfMoney(30M);
-Console.WriteLine(bankAccount4);
+//Console.WriteLine(bankAccount4);
+//bankAccount4.PullOfMoney(50M);
+//bankAccount4.PullOfMoney(30M);
+//Console.WriteLine(bankAccount4);
+
+
+bool result1 = bankAccount3.Transfer(bankAccount4, 20M);
+bool result2 = bankAccount3.Transfer(bankAccount4, 20M);
+
+
+string str1 = "Hello World!";
+Console.WriteLine("Before: {0}, after {1}", str1, Reverse(str1));
+
+
+string path = @"c:\Users\Михаил\source\repos\OOP_Tutorail\Homework2\";
+string inputFile = "inputFile.txt";
+string outputFile = "outputFile.txt";
```

Comment:
```suggestion
const string path = @"c:\Users\Михаил\source\repos\OOP_Tutorail\Homework2\";
const string inputFile = "inputFile.txt";
const string outputFile = "outputFile.txt";
```

Старайтесь по возможности всё, что не изменяется у Вас в программе, делать константами.